app/shared/connectors/httpparsers/HttpParser.scala

```diff
@@ -58,17 +58,24 @@ trait HttpParser extends Logging {
     (__ \ "bvrfailureResponseElement" \ "validationRuleFailures").read[Seq[DownstreamErrorCode]]
   }
 
+  private val singleHipErrorReads: Reads[DownstreamErrorCode] = (__ \ "errorCode").read[String].map(DownstreamErrorCode(_))
+
+  private val multipleHipErrorReads: Reads[Seq[DownstreamErrorCode]] =
+    (__ \ "response" \ "failures").read[Seq[JsObject]].map(_.map(obj => DownstreamErrorCode((obj \ "type").as[String])))
```

Comment:
This can be aligned to the match the `bvrErrorReads` like the below but just as a preference I did a single read.

```
  private val multipleHipErrorReads: Reads[Seq[DownstreamErrorCode]] = {
    implicit val errorTypeReads: Reads[DownstreamErrorCode] = (__ \ "type").read[String].map(DownstreamErrorCode(_))
    (__ \ "response" \ "failures").read[Seq[DownstreamErrorCode]]
  }
```